otherlibs/str/dune

```diff
@@ -0,0 +1,22 @@
+;**************************************************************************
+;*                                                                        *
+;*                                 OCaml                                  *
+;*                                                                        *
+;*                     Thomas Refis, Jane Street Europe                   *
+;*                                                                        *
+;*   Copyright 2018 Institut National de Recherche en Informatique et     *
+;*     en Automatique.                                                    *
+;*                                                                        *
+;*   All rights reserved.  This file is distributed under the terms of    *
+;*   the GNU Lesser General Public License version 2.1, with the          *
+;*   special exception on linking described in the file LICENSE.          *
+;*                                                                        *
+;**************************************************************************
+
+(library
+ (name str)
+ (modes byte)
+ (wrapped false)
```

Comment:
Does this need wrapped to be false? There is only one `str` module in the library currently.

Reply:
I guess it doesn't!
I was on auto-pilot at this point :p